packages/skills/weather_client/behaviours.py

```diff
@@ -36,29 +44,50 @@ def __init__(self, **kwargs):
 
     def setup(self) -> None:
         """Implement the setup for the behaviour."""
-        search_query_empty_model = Query([Constraint("country",
-                                                     ConstraintType("==", "UK"))], model=None)
-        search_request = OEFMessage(oef_type=OEFMessage.Type.SEARCH_SERVICES,
-                                    id=REQUEST_ID,
-                                    query=search_query_empty_model)
+        if self.context.ledger_apis.has_fetchai:
```

Comment:
We don't need this as there won't be ledger transactions. Remove the balance checks.